packages/api-page-builder-so-ddb-es/src/operations/block/index.ts

```diff
@@ -0,0 +1,63 @@
+import WebinyError from "@webiny/error";
+import {
+    BlockStorageOperations,
+    BlockStorageOperationsCreateParams
+} from "@webiny/api-page-builder/types";
+import { Entity } from "dynamodb-toolbox";
+import { BlockDataLoader } from "./dataLoader";
+import { PluginsContainer } from "@webiny/plugins";
+import { createPartitionKey, createSortKey } from "./keys";
+
+const createType = (): string => {
+    return "pb.block";
+};
+
+export interface CreateBlockStorageOperationsParams {
+    entity: Entity<any>;
+    plugins: PluginsContainer;
+}
+export const createBlockStorageOperations = ({
+    entity
+}: CreateBlockStorageOperationsParams): BlockStorageOperations => {
+    const dataLoader = new BlockDataLoader({
+        entity
+    });
+
+    const create = async (params: BlockStorageOperationsCreateParams) => {
+        const { block } = params;
```

Comment:
Quick question - this does contain the ID of the block (`block.id`), right?

Reply:
Yes, ID is created inside `api` create method:
https://github.com/webiny/webiny-js/blob/809fcc5f4a4795d41e69d7fd1b07541e55408046/packages/api-page-builder/src/graphql/crud/blocks.crud.ts#L74

It is used inside "createSortKey"

Reply:
resolving issues